cmd/update.go

```diff
@@ -138,6 +138,9 @@ func getLatestVersion(b *config.Binary, p providers.Provider) (*updateInfo, erro
 	if err != nil {
 		return nil, fmt.Errorf("Error checking updates for %s, %w", b.Path, err)
 	}
+	if v == "" && u == "" {
```

Comment:
In which case is it possible, that we do not have an error but still `v` and `u` are empty?

While reviewing this code, it was not helpful, that the two variables are named `v` and `u`, because this does not tell me anything what I can expect to be in these variables. Also the signature of `GetLatestVersion` does not tell me more (just that I get 2 strings and an error as return value). Therefore I propose to use some more speaking variable names.